schedule/yast/sle/flows/default_ppc64le.yaml

```diff
@@ -0,0 +1,57 @@
+---
+# This is the default ordered sequence of steps
+# to be optionally overwritten for yast ppc64le
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+access_beta: []
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+disk_activation: []
+registration:
+  - installation/registration/register_via_scc
+extension_module_selection:
+  - installation/module_registration/skip_module_registration
+system_probing: []
+add_on_product:
+  - installation/add_on_product/skip_install_addons
+add_on_product_installation: []
+system_role:
+  - installation/system_role/accept_selected_role_text_mode
+guided_partitioning: []
+expert_partitioning: []
+guided_filesystem: []
+suggested_partitioning:
+  - installation/partitioning/accept_proposed_layout
+clock_and_timezone:
+  - installation/clock_and_timezone/accept_timezone_configuration
+local_user:
+  - installation/authentication/use_same_password_for_root
+  - installation/authentication/default_user_simple_pwd
+software: []
+booting:
+  - installation/bootloader_settings/disable_plymouth
+  - installation/bootloader_settings/disable_boot_menu_timeout
+security: []
+default_systemd_target:
```

Comment:
we don't need this item, we have been using:
```
software:
    - installation/installation_settings/validate_default_target
```

Reply:
I was referring to the overwrite not the default, sorry if my comment wasn't clear.